source/platforms/brev-dev.md

```diff
@@ -0,0 +1,82 @@
+## Section 1: Setting up your instance
+
+Navigate to the Brev.dev console and click on "Create your first instance"
+
+![Alt Text](/_static/images/platforms/brev/brev1.png)
+
+Select Container Mode
+
+![Alt Text](/_static/images/platforms/brev/brev2.png)
+
+Attach NVIDIA RAPIDS Container
+
+![Alt Text](/_static/images/platforms/brev/brev3.png)
+
+Configure your own instance
+
+![Alt Text](/_static/images/platforms/brev/brev4.png)
+
+And hit Deploy
+
+![Alt Text](/_static/images/platforms/brev/brev5.png)
+
+## Section 2: Accessing your instance
+
+To simply create and use a Jupyter Notebook:
```

Comment:
This comes up a lot on technical writing review, it's good to avoid words like "simply", "easy", "trivial", etc. It implies a judgement on someone who is stuck and is not finding things simple.

```suggestion
To create and use a Jupyter Notebook:
```

Further reading:
- https://justsimply.dev/
- https://blog.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/